medical/models/medical_patient.py

```diff
@@ -95,6 +95,15 @@ def action_invalidate(self):
             ('z', 'law marriage'),
         ])
 
+    @api.multi
+    def onchange_state(self, state_id):
+        '''
+        Hack to allow for onchange_state on partner.
+        Uses current user's partner, as there should be no correlation
+        between the partner and the response from this method anyways
+        '''
+        return self.env.user.partner_id.onchange_state(state_id)
```

Comment:
Yeah nvm it doesn't seem to actually update the country if I use the api.onchange, unless I'm doing something wrong. 

No errors or anything, just no values updated in the forms. Below is what I have, return is below that.

```
    @api.multi
    @api.onchange('state_id')
    def onchange_state(self):
        """ Hack to allow for onchange_state on partner.
        Uses current user's partner, as there should be no correlation
        between the partner and the response from this method anyways
        """
        return self.env.user.partner_id.onchange_state(self.state_id.id)
```